scripts.md

```diff
@@ -126,6 +126,20 @@ mkdir -p example/en_app/outputs/dialog
 python scripts/dialog_json2xml.py example/en_app/outputs/dialogNew.json -d example/en_app/outputs/dialog/ -v
 ```
 
+## Add additional json to the workspace
```

Comment:
Please update this to reflect non-positional arguments.

Reply:
should be fixed now

Reply:
Thank you. Please update also the corresponding text.

Reply:
fixed